cle/backends/pe.py

```diff
@@ -55,35 +57,33 @@ def relocate(self, solist, bypass_compatibility=False):
                 # no work required
                 pass
             elif self.reloc_type == pefile.RELOCATION_TYPE['IMAGE_REL_BASED_HIGHLOW']:
-                org_bytes = ''.join(self.owner_obj.memory.read_bytes(self.addr, 4))
+                org_bytes = ''.join(self.owner_obj.memory.read_bytes(
+                    AT.from_lva(self.addr, self.owner_obj).to_rva(), 4))
                 org_value = struct.unpack('<I', org_bytes)[0]
-                rebased_value = org_value + self.owner_obj.rebase_addr - self.owner_obj.requested_base
-                rebased_bytes = struct.pack('<I', rebased_value % 2**32)
-                self.owner_obj.memory.write_bytes(self.dest_addr, rebased_bytes)
+                rebased_value = AT.from_lva(org_value, self.owner_obj).to_mva()
+                rebased_bytes = struct.pack('<I', rebased_value % 2 ** 32)
```

Comment:
I think our current style is to not put spaces around the `**` operator when it's just 2 to the power something. Makes it more readable imo but if you feel strongly about it it can stay it the codebase as a statement :)